examples/deployment/kubernetes/README.md

```diff
@@ -0,0 +1,69 @@
+Deploying onto Kubernetes in Google Cloud
+=========================================
+
+This document guides you through the process of spinning up an example Trillian
+deployment on Google Cloud using Kubernetes and Cloud Spanner.
+
+Prerequisites
+-------------
+
+1. You should have this repo checked out :)
+1. A recent debian distribution (other platforms may work, but YMMV)
+1. You must have the `jq` binary installed
+1. You have a Google account with billing configured
+
+Process
+-------
+1. Go to [Google Cloud Console](https://console.cloud.google.com)
+1. Create a new project
+1. Edit the [config.sh](config.sh) file, set `PROJECT_NAME` to the name of your
+   project
+1. run: `./create.sh`.
+   This script will create the Kubernetes cluster, node pools, and Spanner
+   database, service account and etcd cluster.
+   It should take about 5 to 10 minutes to finish and must complete without
+   error.
+1. Now you can deploy the trillian services.
+   Run: `./deploy.sh`
+   This will build the trillian docker images, tag them, and create/update the
+   Kubernetes deployment.
+1. To update a running deployment, simply re-run `./deploy.sh` at any time.
+
+Next steps
+----------
+You should now have a working Trilian Log deployment in Kubernetes.
+To do something useful with it, you'll need to run a "personality" layer, and
+provision one or more trees into the Trillian log.
```

Comment:
Maybe reverse the order of these two clauses to match the order of the text below.

Reply:
done